### ISSUE TITLE
Fix images squished in some orientations

### ISSUE THREAD

**Issue by pirate** [pirate/pennybags#203]
In a few areas of the site, the user profile icons are skewed/squashed into being square, when the original images arent.

**Comment by yamijuan** [comment]
Hi @pirate can we get an screenshot of the problem when you can? thanks

**Comment by pirate** [comment]
- https://pushforward.nyc/leases/
  ![image](https://user-images.githubusercontent.com/511499/88597597-93383580-d035-11ea-831d-1c4d1b05087b.png)

- https://pushforward.nyc/leases/detail/045ad072-86d2-4834-805d-0770eeb8075a
  ![image](https://user-images.githubusercontent.com/511499/88597644-af3bd700-d035-11ea-91f8-fb6a215dc7e4.png)

- https://pushforward.nyc/leases/client/7601a459-0dc8-4c91-8cd9-42ddc950391e
  ![image](https://user-images.githubusercontent.com/511499/88597840-25403e00-d036-11ea-8c98-503fc7409571.png)

- homepage hover on map pins
  ![Screen Shot 2020-08-24 at 10 24 19 PM](https://user-images.githubusercontent.com/511499/91115659-aea85780-e658-11ea-94f5-d093c66434b2.png)



Basically anywhere you have hardcoded both width and height like this is broken:
```html
<img class="rounded-circle img-fluid mx-auto" width="75" height="75" src="/static/images/blank-profile-picture.png" alt="Missing" title="" style="">
```
